server/services/reportService.test.ts

```diff
@@ -76,19 +76,19 @@ describe('ReportService', () => {
         reportForProbationRegionFilename as jest.MockedFunction<typeof reportForProbationRegionFilename>
       ).mockReturnValue('some-filename')
 
-      const month = '1'
-      const year = '2023'
-      const type: ReportType = 'occupancy'
+      const startDate = '01/02/2024'
```

Comment:
should these `/`s be `-`s? Not that it will effect functionality

Reply:
Yes, these should be ISO-formatted -- good call!